charts/deepgram-self-hosted/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 ## [Unreleased]
 
+### Changed
+
+- AWS samples updated to take advantage of new [EKS accelerated AMIs](https://github.com/awslabs/amazon-eks-ami/releases/tag/v20240928), which bundle the required NVIDIA driver and toolkit instead of being installed by the NVIDIA GPU operator 
+
```

Comment:
Also, do any of these [setup instructions](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html) apply such that we need to add more configuration steps to the Helm chart in some way?

Reply:
Those are manual instructions of pieces that are required, but are actually already handled by the `gpu-operator` Helm dependency we have.

More detailed description, the updated `values.yaml` file in this PR enables the `gpu-operator`, but specifically disables the driver and toolkit installation. Well, you may ask, what else is installed then if those two aren't installed? Answer is: the main thing is the NVIDIA device plugin. So our `gpu-operator` dependency handles those additional setup setps.